docs/manual/templates.rst

```diff
@@ -488,7 +488,8 @@ Variabele                              Beschrijving
 ``{{ submission.pdf_url }}``           De URL van het inzendingsrapport (in PDF formaat) in de documenten API.
 ``{{ submission.csv_url }}``           De URL van het inzendingsrapport (in CSV formaat) in de documenten API. Dit document is mogelijk niet aangemaakt
 ``{% json_summary %}``                 JSON met ``"<variabele-eigenschapsnaam>": "<waarde>"`` van alle formuliervelden.
-``{% uploaded_attachment_urls %}``     Een lijst met de URLs van documenten toegevoegd door de inzender. De URLs verwijzen naar het geregistreerde document in de Documenten API.
+``{% as_json "path.to.data.0" %}``     TODO in Dutch
```

Comment:
this is not what we discussed, the argument to `as_json` should not be a string. It should just be a reference to the template variable:

```suggestion
``{% as_json path.to.data.0 %}``     Sluit de gerefereerde variabele (`path.to.data.0`) in als JSON.
```